tests/positive/general.go

```diff
@@ -0,0 +1,63 @@
+// Copyright 2017 CoreOS, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package positive
+
+import (
+	"testing"
+
+	"github.com/coreos/init/tests/register"
+)
+
+func init() {
+	register.Register(register.Test{
+		Name: "Base Test",
+		Func: baseTest,
+	})
+}
+
+func baseTest(t *testing.T, test register.Test) {
```

Comment:
Assuming this is meant as an exemplar test, most of it should be moved into generic code where possible.  See below.